ci-operator/config/openshift/os/openshift-os-master__periodic.yaml

```diff
@@ -124,11 +131,22 @@ resources:
       cpu: 2000m
       memory: 4Gi
 tests:
-- as: validate-machine-os-oci-image
-  commands: cat /etc/os-release
-  container:
-    from: machine-os-oci-content
+- as: validate-built-images
   cron: '@daily'
+  steps:
+    test:
+    - as: validate-rhcos-base-image
```

Comment:
Note to reviewer: I combined these under a single test so we have a single job that builds both images. Otherwise, the image builds could run at different times and may get out of sync.